common/components/BalanceSidebar/Promos.scss

```diff
@@ -1,15 +1,34 @@
-@import "common/sass/variables";
-@import "common/sass/mixins";
+@import 'common/sass/variables';
+@import 'common/sass/mixins';
 
 .Promos {
+  overflow: hidden;
+
+  &-promo-wrapper {
+    height: 6rem;
+    overflow: hidden;
+  }
+
+  &-Bity {
+    background-color: #006e79;
+  }
+
+  &-Coinbase {
+    background-color: #2b71b1;
+  }
+
+  &-HardwareWallets {
+    background-color: #6e9a3e;
+  }
+
   &-promo {
     position: relative;
-    height: 6rem;
+    height: inherit;
     display: block;
     color: #fff;
     text-decoration: none;
     text-align: center;
-    transition-duration: 200ms;
+    transition: opacity 200ms;
```

Comment:
You can just kill this if it's being handled in `.carousel-exit`

Reply:
the transition here is for `:hover` effect